config/config.go

```diff
@@ -105,6 +106,8 @@ func Get() *SourcesApiConfig {
 	options.SetDefault("MarketplaceHost", os.Getenv("MARKETPLACE_HOST"))
 	options.SetDefault("SlowSQLThreshold", 2) //seconds
 	options.SetDefault("BypassRbac", os.Getenv("BYPASS_RBAC") == "true")
+	// We assume that if the "VAULT_ADDR" environment variable is set, Vault is online
+	options.Set("VaultOn", os.Getenv("VAULT_ADDR") != "")
```

Comment:
I'm not completely in love with this - **but we can probably leave it for now, it's just the naming**.

---

But I'm wondering if instead we could have an environment variable that was `"SECRET_STORE"` (or something) and then have convenience methods on the config struct

OR just have config package level functions that look and see if the store is configured to be vault or not. e.g.

```go
func VaultOn() bool { 
	return config.SecretStore == "vault"
}
```

We'd probably default to Postgres as well - which is what we're doing here.

Reply:
Yeah, I like this idea more. I'm changing it!